packages/cx-flowcontainer/doc/FlowContainer.md

```diff
@@ -1,21 +1,31 @@
-# Vital FlowContainer Tokens
+# Vital Flow Container Tokens
 
-The Vital flowcontainer is just a pass-thru and uses [Bodiless Flowcontainer](@jones link to bodiless #/Components/FlowContainer/).  The vital flowcontainer package does provides some useful tokens that can be used to quick constrainments on flowcontainer.  It also provides token that provides ability to do a ContentRegion aka. nested flowcontainer.
+The Vital Flow Container is just a passthrough, and uses the [Bodiless Flow
```

Comment:
@misterjones  should we remove "just" here?   I often put them it to make it sound simple and then remove because I believe its little like talking down to reader?

Reply:
I agree, I try to remove those as well. This one slipped through!